applications/DEMApplication/tests/test_DEM_3D_continuum_vs_discontinuum.py

```diff
@@ -0,0 +1,99 @@
+import os
+import KratosMultiphysics
+from KratosMultiphysics import Logger
+Logger.GetDefaultOutput().SetSeverity(Logger.Severity.WARNING)
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.DEMApplication.DEM_analysis_stage
+
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+import auxiliary_functions_for_tests
+
+this_working_dir_backup = os.getcwd()
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class DEM3D_ContinuumTestVsDiscontinuumSolution(KratosMultiphysics.DEMApplication.DEM_analysis_stage.DEMAnalysisStage, KratosUnittest.TestCase):
+
+    def InitializeSolutionStep(self):
+        super().InitializeSolutionStep()
+
+        self.tolerance = 2.5e-5
```

Comment:
can this tolerance be decreased? I was hoping the error would be smaller

Reply:
OK, I'll recheck!